src/uct/ib/dc/dc_mlx5_ep.h

```diff
@@ -481,6 +481,11 @@ uct_dc_mlx5_iface_dci_get(uct_dc_mlx5_iface_t *iface, uct_dc_mlx5_ep_t *ep)
     ucs_assert(!iface->super.super.config.tx_moderation);
 
     if (uct_dc_mlx5_iface_is_dci_rand(iface)) {
+        /* Coverity wrongly assumes that the endpoint could check TX DCI
+         * resources on IFACE without DCI held, but DCI random policy guarantees
+         * that DCI is always set in the ep->dci */
```

Comment:
maybe shorter:
/* Silence Coverity - in random policy the endpoint always has an assigned DCI */